cases/deposit.test.js

```diff
@@ -77,7 +78,8 @@ describe("Deposit", () => {
       toml: toml,
       isDeposit: true,
     });
-    expect(status).not.toEqual(200);
+    expect(status).toBeGreaterThanOrEqual(400);
+    +expect(status).toBeLessThan(500);
```

Comment:
whats with the plus here

Reply:
Ahh I was copying from a `git diff`, I'll fix that